### PR TITLE
Makes Dogtags removable on death immediately

### PR DESCRIPTION
**Pull request by GriffinMan4455**
# About the pull request

Lets marines take the identifier off a dead buddie's tags immediately on their demise

Tested, works fine

Remember: something that is self-evident to you might not be to others. Explain your rationale fully, even if you feel it goes without saying. -->

# Explain why it's good for the game

Stinky PvP thing where you can only remove them after five minutes since they have like revives and stuff but we don't have that

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
qol: Dogtags removable after 30 seconds
/:cl:

### CODE REVIEW

**Review by DexterDude — APPROVED** [review]

File: code/modules/mob/living/carbon/human/life.dm [hunk 1]
```diff
@@ -63,7 +63,7 @@
 		else //Dead
 			if(!undefibbable)
 				handle_necro_chemicals_in_body(delta_time) //Specifically for chemicals that still work while dead.
-				if(life_tick > 5 && timeofdeath && (timeofdeath < 5 || world.time - timeofdeath > revive_grace_period) && !issynth(src)) //We are dead beyond revival, or we're junk mobs spawned like the clowns on the clown shuttle
+				if(life_tick > 0.5 && timeofdeath && (timeofdeath < 0.5 || world.time - timeofdeath > revive_grace_period) && !issynth(src)) //We are dead beyond revival, or we're junk mobs spawned like the clowns on the clown shuttle
```
Comment (Doubleumc) [inline code/modules/mob/living/carbon/human/life.dm:66]:
```suggestion
				if(life_tick > 5 && timeofdeath && (timeofdeath < 5 || world.time - timeofdeath > revive_grace_period) && !issynth(src)) //We are dead beyond revival, or we're junk mobs spawned like the clowns on the clown shuttle
```
This change doesn't help w/r/t dogtags.

File: code/modules/mob/living/carbon/human/human_defines.dm [hunk 2]
```diff
@@ -84,7 +84,7 @@
 	var/oxygen_alert = 0
 	var/fire_alert = 0
 	var/prev_gender = null // Debug for plural genders
-	var/revive_grace_period = 5 MINUTES //5 minutes
+	var/revive_grace_period = 30 SECONDS //30 Seconds
```
Comment (Doubleumc) [inline code/modules/mob/living/carbon/human/human_defines.dm:87]:
```suggestion
	var/revive_grace_period = 5 MINUTES //5 minutes
```
While this *works*, I'd suggest removing the undefibbable check for dogtags altogether.

You can find the checks in `human_stripping.dm`:
https://github.com/cmss13-devs/cmss13-pve/blob/1ad2ce94ac72293a10b3cae710b04e87a99688b7/code/modules/mob/living/carbon/human/human_stripping.dm#L212
https://github.com/cmss13-devs/cmss13-pve/blob/1ad2ce94ac72293a10b3cae710b04e87a99688b7/code/modules/mob/living/carbon/human/human_stripping.dm#L227

Removing the `!sourcemob.undefibbable` bit means it no longer cares if the subject is revivable or not.

**Review by Doubleumc — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/human_defines.dm (hunk 2 above)
Comment (GriffinMan4455) [reply code/modules/mob/living/carbon/human/human_defines.dm]: I was gonna do this but updated from master, so I tried doing it locally but it...didn't *seem* to work? I'll push what I did and maybe I just SUCK, but I thought I did it right at least

File: code/modules/mob/living/carbon/human/human_stripping.dm [hunk 3]
```diff
@@ -209,7 +209,7 @@ GLOBAL_LIST_INIT(strippable_human_items, create_strippable_list(list(
 	var/mob/living/carbon/human/sourcemob = source
 	if (!istype(tag))
 		return
-	if (!sourcemob.undefibbable && (!skillcheck(user, SKILL_POLICE, SKILL_POLICE_SKILLED) || sourcemob.stat != DEAD))
+	if (!skillcheck(user, SKILL_POLICE, SKILL_POLICE_SKILLED) || sourcemob.stat != DEAD)
```
Comment (Doubleumc) [inline code/modules/mob/living/carbon/human/human_stripping.dm:212]:
```suggestion
	if (sourcemob.stat != DEAD)
```

File: code/modules/mob/living/carbon/human/human_stripping.dm [hunk 4]
```diff
@@ -224,7 +224,7 @@ GLOBAL_LIST_INIT(strippable_human_items, create_strippable_list(list(
 		return
 	if(!istype(sourcemob.wear_id, /obj/item/card/id/dogtag))
 		return
-	if (!sourcemob.undefibbable && !skillcheck(user, SKILL_POLICE, SKILL_POLICE_SKILLED))
+	if ( skillcheck(user, SKILL_POLICE, SKILL_POLICE_SKILLED))
 		return
```
Comment (Doubleumc) [inline code/modules/mob/living/carbon/human/human_stripping.dm:228]:
```suggestion
```

**Review by Doubleumc — CHANGES_REQUESTED** [review]

**Comment by GriffinMan4455** [comment]
You the goat UMC, everything works perfectly now! Tags are immediately grabbable on death o7

**Review by Doubleumc — APPROVED** [review]

**Pull request merged**